dist/es5src/drivers/indexeddb.js

```diff
@@ -0,0 +1,576 @@
+(function (global, factory) {
+    if (typeof define === 'function' && define.amd) {
+        define('asyncStorage', ['exports', 'module'], factory);
+    } else if (typeof exports !== 'undefined' && typeof module !== 'undefined') {
+        factory(exports, module);
+    } else {
+        var mod = {
+            exports: {}
+        };
+        factory(mod.exports, mod);
+        global.asyncStorage = mod.exports;
```

Comment:
It looks like they had a regression on v5.6.8. I will take a look at it
later today.

On Tue, Jul 14, 2015, 02:26 Matthew Riley MacPherson <
notifications@github.com> wrote:

> In dist/es5src/drivers/indexeddb.js
> <https://github.com/mozilla/localForage/pull/390#discussion_r34523581>:
>
> > @@ -0,0 +1,576 @@
> > +(function (global, factory) {
> > +    if (typeof define === 'function' && define.amd) {
> > +        define('asyncStorage', ['exports', 'module'], factory);
> > +    } else if (typeof exports !== 'undefined' && typeof module !== 'undefined') {
> > +        factory(exports, module);
> > +    } else {
> > +        var mod = {
> > +            exports: {}
> > +        };
> > +        factory(mod.exports, mod);
> > +        global.asyncStorage = mod.exports;
>
> Fair enough. I can even do that in a separate PR after this. It's less
> important, you're right [image: :smile:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/localForage/pull/390/files#r34523581>.
>